### PR TITLE
feat(vc): add meeting screenshot shortcut for visual context

### PR DESCRIPTION
**Pull request by Ren1104**
## Summary

Adds `vc +meeting-screenshot` to save the current final-composite frame of an eligible ongoing meeting as a JPEG. The command is intended for tasks that require visual context from the current meeting frame, while the meeting Skill routes agents to structured events, subtitles, chat, or readable shared documents whenever those are sufficient.

## Changes

- Add the `vc +meeting-screenshot` shortcut for user and bot identities using a long `meeting_id`.
- Save screenshots through invocation-scoped FileIO, with a visible default path under `meeting-screenshots/`, explicit `--output` support, and overwrite protection.
- Validate the response as an `image/jpeg` with an 8 MiB limit, and return the absolute path, size, SHA-256 digest, content type, and server `log_id`.
- Add progressive Skill routing under the existing live-meeting interaction scene:
  - use screenshots only when the answer depends on pixels in the current composite frame, such as a displayed URL, UI state, error, chart, slide layout, or camera view;
  - do not screenshot when meeting events, subtitles, chat, or a directly readable shared document can answer the request;
  - keep the existing quick action for querying meeting content unchanged.
- Add unit coverage, dry-run E2E coverage, and a Skill contract test for the routing boundaries.

## Test Plan

- [x] `go test ./shortcuts/vc ./tests/cli_e2e/vc -count=1`
- [x] `node scripts/skill-format-check/index.js`
- [x] `go vet ./...`
- [x] `make fmt-check`
- [x] `QUALITY_GATE_CHANGED_FROM=origin/main make quality-gate`
- [x] `go mod tidy` leaves `go.mod` and `go.sum` unchanged
- [ ] `make unit-test` is blocked before package assertions by a Go 1.26.1 runtime SIGSEGV when `-race` is combined with `-gcflags='all=-N -l'`; plain focused tests and race-only reproduction pass.
- [ ] Live meeting verification was not rerun from the current PR head.

## Related Issues

- None

## Summary by CodeRabbit

- **New Features**
  - Added the `vc +meeting-screenshot` command to retrieve and save current meeting screenshots as JPEG files.
  - Supports custom output paths, timestamped filenames, dry runs, and optional overwrite behavior.
  - Reports file metadata, including size, content type, SHA-256 checksum, and service log ID.
  - Added guidance for using screenshots during live meeting interactions.

- **Bug Fixes**
  - Added validation for meeting IDs, output paths, JPEG responses, and existing files.
  - Preserves existing files when screenshot retrieval or validation fails.
  - Improved handling of service errors and invalid screenshot responses.

### CODE REVIEW

File: skills/lark-meeting/references/lark-vc-meeting-screenshot.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+```bash
+lark-cli vc +meeting-screenshot --as user --meeting-id <long_meeting_id>
+lark-cli vc +meeting-screenshot --as bot --meeting-id <long_meeting_id> --output ./current.jpg --overwrite
+```
+
+- `--meeting-id` 必须是长数字会议 ID，不接受 9 位会议号。
+- 身份必须沿用发现该 `meeting_id` 时使用的身份；用户身份需要当前用户在会中，应用身份需要机器人具备会中读取权限。
+- 服务端会校验会议是否满足截图条件；调用方不支持指定截图内容。
+- 默认写入 `.lark-vc/screenshots/<meeting_id>-<UTC timestamp>.jpg`；已有目标文件时默认失败，只有 `--overwrite` 可以替换。
```
Comment (zhaoleibd) [inline skills/lark-meeting/references/lark-vc-meeting-screenshot.md]: 路径改一下

File: skills/lark-meeting/SKILL.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 ---
 name: lark-meeting
 version: 1.0.0
-description: "飞书视频会议：查询会议记录与会议产物(纪要/逐字稿/妙记)、妙记搜索/上传/下载/编辑、机器人参与会议；查询进行中的会议、实时会议内容(发言/聊天/共享文档)问答(会上/会里)、发送会中聊天/表情；基于 meeting_id、meeting_no、event_id、note_id、minute_token、vc-node-id 或妙记 URL 查询相关信息。预约会议、忙闲和会议室管理走 lark-calendar。"
+description: "飞书视频会议：查询会议记录与会议产物(纪要/逐字稿/妙记)、妙记搜索/上传/下载/编辑、机器人参与会议；查询进行中的会议、实时会议内容(发言/聊天/共享文档)问答(会上/会里)、获取视频会议截图、发送会中聊天/表情；基于 meeting_id、meeting_no、event_id、note_id、minute_token、vc-node-id 或妙记 URL 查询相关信息。预约会议、忙闲和会议室管理走 lark-calendar。"
```
Comment (zhaoleibd) [inline skills/lark-meeting/SKILL.md:4]: 感觉不需要加到 description 里

**Comment by Ren1104** [comment]
Merged in another PR
